tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Item Create.feature

```diff
@@ -0,0 +1,134 @@
+﻿Feature: Create order item
+    As a Buyer User
+    I want to create an order item for a given order
+    So that I can complete my order
+
+Background:
+    Given Orders exist
+        | OrderId    | Description      | OrganisationId                       | LastUpdatedByName | LastUpdatedBy                        | CommencementDate | CatalogueSolutionsViewed |
+        | C000014-01 | Some Description | 4af62b99-638c-4247-875e-965239cd0c48 | Tom Smith         | 335392e4-4bb1-413b-9de5-36a85c9c0422 | 01/01/2021       | false                    |
+    And Service Recipients exist
+        | OdsCode | Name              | OrderId    |
+        | ODS1    | Service Recipient | C000014-01 |
+    And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+
+@7840
+Scenario: 1. Create catalogue solution order item
```

Comment:
Could this scenario description be improved as it looks like the test also creates an additional service order item as well. How about just "Create order item"? Could this be applied to the rest of the file?

Reply:
Good spot